api/common/models/project.js

```diff
@@ -125,13 +125,13 @@ module.exports = (Project) => {
         ({ totalEstimate, totalActual }, currentProjectTask) => {
           projectTaskIds.push(currentProjectTask.id)
           return {
-            totalEstimate: totalEstimate + currentProjectTask.estimateTime * 60,
+            totalEstimate: totalEstimate + currentProjectTask.estimateTime,
             totalActual:
               totalActual +
               currentProjectTask
                 .activities()
                 .reduce((total, currentActivity) => {
-                  return total + currentActivity.length
+                  return total + (currentActivity.length / 60)
```

Comment:
<img width="322" alt="CleanShot 2023-08-16 at 21 31 01@2x" src="https://github.com/joshuawootonn/time-track/assets/20950876/39316d8d-eac4-4090-b55f-b232e47da79e">


On second thought, I think all these lengths should be in minutes. Then we can format them any way we like on the frontend.

Reply:
When you do these types of divisions make sure you `Math.floor`. We don't want to have to handle any wild decimal places on the FE.